src/lib/y2network/wicked/connection_config_writers/base.rb

```diff
@@ -105,6 +106,13 @@ def add_hostname(conn)
 
           Yast::Host.Update("", conn.hostname, conn.ip.address.address.to_s)
         end
+
+        # @param conn [Y2Network::ConnectionConfig::Base] Connection to take settings from
+        def static_ip?(conn)
+          return false unless conn.bootproto.static?
+
+          conn.ip && conn.ip.address.address.to_s != "0.0.0.0"
```

Comment:
this method does little bit more than described ... it is not only about checking whether it is static ip or not as method name suggests.

Reply:
Well, if the bootproto is not static then it reports false as although there is an static IP it is irrelevant, and in case of static IP it should be a valid one, it is valid according to **IPAddr** but not a valid config to be written, valid_static_ip? sounds better to you? what you propose, it is a convenience method so not exposed at all.

Reply:
yeah sounds better ... 

code should make things as clear as possible. Doesn't depend if the method is public or not. Especially now when functionality is "hidden" (nested) in a hierarchy of classes.